### PR TITLE
Added /operations/database endpoint. #96

### PR DESCRIPTION
**Pull request by WalterMoar**
I split the database check out from the regular health check, as we had pod restarts due to db timeouts.

### CODE REVIEW

File: ppr-api/src/endpoints/healthcheck.py [hunk 1]
```diff
@@ -15,12 +15,20 @@
 @router.get("/health")
 def health(session: sqlalchemy.orm.Session = fastapi.Depends(models.database.get_session)):
```
Comment (jguertin) [inline ppr-api/src/endpoints/healthcheck.py]: If not using the database in `/health`, you can remove this parameter.

**Review by jguertin — APPROVED** [review]
lgtm, just one comment on the health method parameter.  If you like we can remove this later, but would prefer sooner.

File: ppr-api/src/endpoints/healthcheck.py (hunk 1 above)
Comment (WalterMoar) [reply ppr-api/src/endpoints/healthcheck.py]: Oopsies.

Comment (WalterMoar) [reply ppr-api/src/endpoints/healthcheck.py]: Removed.

**Pull request merged**